### PR TITLE
added favicon, added hero image

### LINKED ISSUE #102: Add hero banner and favicon
I think just need drop favicon.ico to `/public` folder and that's it

image

plus add hero banner on frontpage would make it more impressive I guess

### PR DESCRIPTION
**Pull request by poppyseedDev**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #102 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [ ] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

## Copilot Summary
copilot:summary

copilot:poem

### CODE REVIEW

File: developers/index.md [hunk 1]
```diff
@@ -5,7 +5,10 @@ layout: home
 hero:
   name: "KodaDot "
   text: "developers guide"
-  tagline: Open Source Multichain NFT Marketplace
+  tagline: An Open Source Polkadot NFT Marketplace
+  image:  
+    src: /devs-cover.png
+    alt: SubWork web3 jungle
```
Comment (yangwao) [inline developers/index.md:11]: what is this 😅

File: developers/index.md [hunk 2]
```diff
@@ -28,5 +31,11 @@ features:
     details: Ultimate super charged search to search multi-chain NFT marketplaces
     link: /polysearch/index
 
+head:
+  - - meta
+    - name: description
+      content: Kodadot is a nuber 1 Polkadot NFT Marketplace that aims to be the best ...
+  - - meta
+    - name: keywords
+      content: Polkadot NFT Marketplace, NFTs, KodaDot, alha D1 Cloudflare Workers, serverless, distributed, API, GET request, POST request, search endpoint, search table, search query, chain, limit, offset, install wrangler, dependencies, initialize database, local server, worker, localhost:8787
```
Comment (yangwao) [inline developers/index.md:40]: localhost:8787 is someone looking for? I think some keywords are turbo random

**Review by yangwao — COMMENTED** [review]
few keywords need changed tbh
